backend/main-service/src/main/java/kkakka/mainservice/order/ui/OrderController.java

```diff
@@ -1,40 +1,42 @@
 package kkakka.mainservice.order.ui;
 
-import kkakka.mainservice.member.application.MemberService;
 import kkakka.mainservice.order.application.OrderService;
-import kkakka.mainservice.order.ui.dto.OrderRequestDto;
-import kkakka.mainservice.product.application.ProductService;
+import kkakka.mainservice.order.ui.dto.OrderRequest;
+import kkakka.mainservice.order.ui.dto.OrderResponse;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 import java.net.URI;
+import java.util.List;
 
 @RestController
 @RequestMapping("/orders")
 public class OrderController {
 
     private final OrderService orderService;
-    private final MemberService memberService;
-    private final ProductService productService;
 
-    public OrderController(OrderService orderService, MemberService memberService, ProductService productService) {
+    public OrderController(OrderService orderService) {
         this.orderService = orderService;
-        this.memberService = memberService;
-        this.productService = productService;
     }
 
     @GetMapping
-    public void createForm(Model model) {
+    public void showOrderFormInfo(
+        //TODO: memberId -> 회원 정보랑 장바구니 목록을 전부 return 해줌
+    ) {
     }
 
     @PostMapping
-    public ResponseEntity<Void> order(@RequestBody OrderRequestDto orderRequestDto) throws Exception {
-
-        Long memberId = orderService.order(orderRequestDto);
+    public ResponseEntity<Void> order(@RequestBody OrderRequest orderRequest) throws Exception {
```

Comment:
```suggestion
    public ResponseEntity<Void> order(@RequestBody OrderRequest orderRequest) {
```

내부에 딱히 checked exception 없는거 같은데 확인해주세요!